src/mlpack/methods/decision_tree/gini_gain.hpp

```diff
@@ -39,6 +39,7 @@ class GiniGain
       return 0.0;
 
     CountType impurity = 0.0;
+    #pragma omp parallel for reduction(+:impurity)
     for (size_t i = 0; i < countLength; ++i)
```

Comment:
For some reason you can't use `size_t` + OpenMP + MSVC, so we created an alias `omp_size_t` that works across Linux, MacOS and Windows.

Reply:
So I will have to replace size_t with omp_size_t every time i try to use it in parallel parts of my program?